src/theme/Tooltip/tooltip.js

```diff
@@ -98,11 +98,13 @@ class Tooltip extends React.Component {
           tooltipClassName,
         ].join(' ')}
         >
-          <span className={`${styles.tooltipArrow} tooltip-arrow`}>
-            <svg stroke="inherit" fill="currentColor" viewBox="0 0 14 28">
-              <path d="M13.307.5S.5 10.488.5 13.896c0 3.409 12.785 12.893 12.785 12.893" />
-            </svg>
-          </span>
+          {!this.props.noArrow && (
```

Comment:
```suggestion
          {!this.props?.noArrow && (
```

Reply:
I don't think there is a need for this because this.props would always exists in a class component